### PR TITLE
Document new excludeAttributes array in Password Complexity Object

### PR DESCRIPTION
**Pull request by mpeng-okta**
## Description:
- Add excludeAttributes array in Password Complexity Object documentation

### Resolves:

* [OKTA-140223](https://oktainc.atlassian.net/browse/OKTA-140223)

## Screenshot:
image

### CODE REVIEW

File: _source/_docs/api/resources/policy.md [hunk 1]
```diff
@@ -1148,6 +1148,7 @@ minUpperCase | Indicates if a password must contain at least one upper case lett
 minNumber | Indicates if a password must contain at least one number: 0 indicates no, 1 indicates yes | integer | No | 1
 minSymbol | Indicates if a password must contain at least one symbol (e.g., !@#$%^&*): 0 indicates no, 1 indicates yes | integer | No | 1
 excludeUsername | Indicates if the user name must be excluded from the password | boolean | No | true
+excludeAttributes | The user profile attributes whose attribute values must be excluded from the password: currently only supports `firstName` and `lastName` | Array | No | Empty Array
```
Comment (edjohns-okta) [inline _source/_docs/api/resources/policy.md]: I think no need to have "attributes" and "attribute" in the sentence together.  Could we just say "The user profile attributes whose values must be excluded..."

Comment (mpeng-okta) [reply _source/_docs/api/resources/policy.md]: That's a good point. I had that verbatim originally, but my concern was if people might confused "values" here with the attribute names themselves. If you think it's not a big deal, I'll go ahead and update it.

Comment (mystiberry-okta) [reply _source/_docs/api/resources/policy.md]: Agree w/Ed. You could even say "The user-profile attribute values" or "The values of user profile attributes", but deleting the 2nd "attribute" is fine.

Comment (mpeng-okta) [reply _source/_docs/api/resources/policy.md]: Yeah, I think I'll just drop the 2nd attribute.

**Comment by mpeng-okta** [comment]
Yikes, I messed up big time; I haven't merged the okta-core change in yet. @mystiberry-okta would it be possible to pull this out of the weekly branch for now? Terribly sorry about that.

**Pull request merged**